rocky/rocky/views/mixins.py

```diff
@@ -46,6 +48,19 @@ class OriginData(BaseModel):
     boefje: Boefje | None = None
     params: dict[str, str] | None = None
 
+    @property
+    def is_old(self, maxage: Optional[int] = None) -> bool
+        if not maxage:
+            maxage = ORIGIN_MAX_AGE
+        if not self.normalizer:
+            return False
+            
+        observationdate = self.normalizer.get("raw_data", {}).get("boefje_meta", {}).get("ended_at", False)
```

Comment:
Two things:
- `observartion_date` is a more clear and Pythonic name
- The latest `False` which is used as a default value doesn't make sense. By default it returns `None`, which is a better way of indicating non-existence. Also, try not to mix too much types: Ideally you want at most a `datetime | None`, and not a `datetime | bool | None`.

```suggestion
        observation_date = self.normalizer.get("raw_data", {}).get("boefje_meta", {}).get("ended_at")
```